x/pairing/keeper/scores/stake_req.go

```diff
@@ -0,0 +1,28 @@
+package scores
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+)
+
+const stakeReqName = "stake-req"
+
+// StakeReq implements the ScoreReq interface for provider staking requirement(s)
+type StakeReq struct {
+	MinStake sdk.Int
+}
+
+// Score calculates the the provider score as the normalized stake
+func (sr StakeReq) Score(stakeEntry epochstoragetypes.StakeEntry) uint64 {
+	return stakeEntry.Stake.Amount.Quo(sr.MinStake).Uint64()
```

Comment:
we said to normalize by minStake/16 so we have a more refined quantization, currently:
5000000 and 999999 will have the same chance

Reply:
done. see https://github.com/lavanet/lava/commit/fa97b594033898263bad056fd6fde790cca859c3